lib/commands/show-asset-sizes.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path    = require('path');
+var Command = require('../models/command');
+
+module.exports = Command.extend({
+  name: 'show-asset-sizes',
+  description: 'Output asset file sizes.',
+
+  availableOptions: [
+    { name: 'asset-path', type: path, default: 'dist/assets' }
```

Comment:
we may want this to come from --output-path, as users can configure this in there .emberrc